hw3/js/hw3-4.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const firstNumber = prompt('Enter the first number', '');
```

Comment:
you can combine methods)
```suggestion
    const firstNumber = Number(prompt('Enter the first number', ''));
```